recipes/mleap/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "mleap" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: eb384a239747f319297a0b4f2bb5c076f01cfd942d7f72a173494606f402fdad
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - nose-exclude >=0.5.0
```

Comment:
Are all of these needed as host requirements? You probably only need `python` and `pip`.

Reply:
No, they aren't, sorry. Fixed this.